containers/netmhctools/1.1.1/Dockerfile

```diff
@@ -6,6 +6,7 @@ LABEL org.opencontainers.image.vendor="MSKCC-OMICS-WORKFLOWS" \
       org.opencontainers.image.created="2025-08-20T11:07:00Z" \
       org.opencontainers.image.licenses="ACADEMIC SOFTWARE LICENSE AGREEMENT" \
       org.opencontainers.image.version="1.1.1" \
+      imageprivacy="False" \
```

Comment:
shouldn't this be set to true? The code base you're pulling from is supposed to be private. If we make the container not private, doesn't that mean the code base will be available to anyone who can download this container?

also needs to be lowercase `true`.

Reply:
GHCR is going to be used for us to distribute internally, the license should be with the docker image, we will look into ways to prevent unauthorized access